planning/scenario_planning/parking/astar_search/src/astar_search.cpp

```diff
@@ -14,56 +14,58 @@
  * limitations under the License.
  */
 
-#include "astar_search/astar_search.h"
-
-#include <vector>
+#include <astar_search/astar_search.h>
```

Comment:
[Minor]: I think the original usage of `""` instead of `<>` is correct when referencing files local to this project?